components/benefit_card_additional_info.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import ErrorOutlineIcon from "@material-ui/icons/ErrorOutline";
+import Paper from "@material-ui/core/Paper";
+import { logEvent } from "../utils/analytics";
+import { connect } from "react-redux";
+import { css } from "react-emotion";
+
+const cardTop = css`
+  background-color: #f1f7fc;
+  border-radius: 0px;
+  border-bottom: 1px solid #8b8b8b;
+  padding: 15px 15px 15px 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+const parentIcon = css`
+  margin-right: 15px;
+  font-size: 40px !important;
+  transform: scale(0.9);
+  color: #434343;
+`;
+const headerDesc = css`
+  flex-grow: 1;
+  color: #434343;
+`;
+
+export class BenefitCardHeaderMoreInfo extends Component {
+  logExit = url => {
```

Comment:
you don't need the `logExit` function here, it's just used to log when the user clicks on a VAC site url in the other header.

Reply:
@sastels just seeing these comments now, were the files updated?